srcs/user/mod.rs

```diff
@@ -80,6 +80,7 @@ pub unsafe fn exec_fn_userspace(
 	let cr3 = get_paddr!(page_dir as *const _);
 	new_task.regs.cr3 = running_task.regs.cr3;
 	process.test = true;
+	process.owner = 1;
```

Comment:
If the owner is set to 1 when it's a user process and 0 when it's kernel process, then we can remove the member `test` from the Process structure which where there only to differentiate user processes and kernel processes.